src/Core/UbuntuDriversBackend.vala

```diff
@@ -18,6 +18,18 @@
  */
 
 public class AppCenterCore.UbuntuDriversBackend : Backend, Object {
+
+    private bool _working;
+    public bool working {
```

Comment:
Since this is only getting and setting the property, wouldn't it make sense to just do what is in the flatpak backend: `public bool working { get; private set; }`?